nixos/tests/deluge.nix

```
@@ -5,41 +5,6 @@ import ./make-test-python.nix ({ pkgs, ...} : {
   };
 
   nodes = {
-    simple1 = {
-      services.deluge = {
-        enable = true;
-        package = pkgs.deluge-1_x;
-        web = {
-          enable = true;
-          openFirewall = true;
-        };
-      };
-    };
-
-    declarative1 = {
-      services.deluge = {
-        enable = true;
-        package = pkgs.deluge-1_x;
-        openFirewall = true;
-        declarative = true;
-        config = {
-          allow_remote = true;
-          download_location = "/var/lib/deluge/my-download";
-          daemon_port = 58846;
-          listen_ports = [ 6881 6889 ];
-        };
-        web = {
-          enable = true;
-          port =  3142;
-        };
-        authFile = pkgs.writeText "deluge-auth" ''
-          localclient:a7bef72a890:10
-          andrew:password:10
-          user3:anotherpass:5
-        '';
-      };
-    };
-
     simple2 = {
```

you could call that just "simple" now
```suggestion
    simple = {
```

---

Sorry, saw it too late. I will try to think of it the next time we change the test.